docs/Cli.md

```diff
@@ -133,6 +133,7 @@ Re-apply any new defaults as desired.
 | `auto_disarm_delay`             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 0      | 60     | 5             | Master       | UINT8    |
 | `small_angle`                   | If the copter tilt angle exceed this value the copter will refuse to arm. default is 25°.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 180    | 25            | Master       | UINT8    |
 | `pid_at_min_throttle`           | If enabled, the copter will process the pid algorithm at minimum throttle.  Cannot be used when `retarded_arm` is enabled.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | OFF    | ON     | ON            | Master       | UINT8    |
+| `airmode_saturation_limit`      | When air mode is enabled this will act as a "crash detection". It prevents too much compensation for very high errors what can cause spazzing out. 0 = no limit 100 is always limited during saturation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 100    | 50            | Master       | UINT8    |
```

Comment:
Was wrong - output range is 16 bit.

https://github.com/cleanflight/cleanflight/blob/master/src/main/sensors/gyro.c#L128

gyro offset must be applied with saturation - there is overflow with 16bit values.

Also using biquad filter may result in overflow (https://github.com/cleanflight/cleanflight/blob/master/src/main/sensors/gyro.c#L145)

IMO it would be best to either use int32 for gyroADC or to scale gyro reading down (divide by 2 or 4) so that there is some safety margin.